vault/README.md

```diff
@@ -190,6 +190,8 @@ Run the [Agent's status subcommand][13] and look for `vault` under the Checks se
 
 See [metadata.csv][14] for a list of metrics provided by this integration.
 
+**Note**: Version 3.3.0+ of this check uses [OpenMetrics][21] for metric collection, which requires Python 3. For hosts that are unable to use Python 3, or if you would like to use a legacy version of this check, set the value of `use_openmetrics` to `false` in the configuration.
```

Comment:
Small nit- Since this is a breaking change for customers (the V2 version will now be enabled by default), then I think this should be a major version bump (so a 4.0.0 release). To indicate this, you can update the changelog label to `changelog/changed`
```suggestion
**Note**: Version 4.0.0+ of this check uses [OpenMetrics][21] for metric collection, which requires Python 3. For hosts that are unable to use Python 3, or if you would like to use a legacy version of this check, set the value of `use_openmetrics` to `false` in the configuration.
```